backend/middlewares/post-middleware.js

```diff
@@ -11,12 +11,12 @@ export const isAuthorMiddleware = async (req, res, next) => {
     }
 
     console.log(post.authorId, userId);
-    if (post.authorId.toString() !== userId) {
+    if (post?.authorId?.toString() !== userId) {
       return res
```

Comment:
Why do we need  `?`  here? isn't authorId mandatory for the post? ideally we are not supposed to use `?`